package.json

```
@@ -61,5 +61,6 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

```suggestion
  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

This matches [how we have it set over in Forge](https://github.com/electron/forge/blob/38f64e4c9e2c79a3b9ff8447b1a23199415ede97/package.json#L180).

---

Hmm, is that meant to match the sha1 of the vendored Yarn file? I get something different locally, but perhaps I'm hashing the wrong thing?

```
shasum -a 1 .yarn/releases/yarn-1.22.22.cjs
870faaceb1604f213264cfa69f7db5b620d086b2  .yarn/releases/yarn-1.22.22.cjs
```

---

The way we pulled it for Forge was using the output of `npm show yarn@1.22.22` - not sure if a different approach is needed in this case with the file being vendored.

```
$ npm show yarn@1.22.22

yarn@1.22.22 | BSD-2-Clause | deps: none | versions: 120
📦🐈 Fast, reliable, and secure dependency management.
https://github.com/yarnpkg/yarn#readme

bin: yarn, yarnpkg

dist
.tarball: https://registry.npmjs.org/yarn/-/yarn-1.22.22.tgz
.shasum: ac34549e6aa8e7ead463a7407e1c7390f61a6610
.integrity: sha512-prL3kGtyG7o9Z9Sv8IPfBNrWTDmXB4Qbes8A9rEzt6wkJV8mUvoirjU0Mp3GGAU06Y0XQyA3/2/RQFVuK7MTfg==
.unpackedSize: 5.3 MB

maintainers:
- byk <ben@byk.im>
- bestander <bestander@gmail.com>
- daniel15 <npm@d.sb>
- arcanis <nison.mael@gmail.com>
- danbuild <build@dan.cx>

dist-tags:
berry: 2.4.3     latest: 1.22.22  legacy: 1.21.1   rc: 1.22.0       

published 3 months ago by arcanis <nison.mael@gmail.com>
```

---

In this case it might be unnecessary since we're vendoring the file itself.